### PR TITLE
vdk-frontend: update frontend docs

### PR DESCRIPTION
**Pull request by murphp15**
# Why
We want the frontend VEP to describe all major decisions in the implementation. We now include details about docker and helm.

### CODE REVIEW

File: specs/vep-1507-vdk-operations-ui/README.md [hunk 1]
```diff
@@ -138,6 +140,16 @@ supports customers that want to:
 1. Use the shared component libraries in their projects.
 2. Build their own docker image for data-pipelines
 
+#### Docker
+A Docker image will be published on docker hub under the user https://hub.docker.com/u/versatiledatakit.
```
Comment (ivakoleva) [inline specs/vep-1507-vdk-operations-ui/README.md:144]: nit: may rephrase the statement, so we don't need to update it upon making this change -> or alternatively have the ticket linked, so we don't forget to update it?

File: specs/vep-1507-vdk-operations-ui/README.md [hunk 2]
```diff
@@ -138,6 +140,16 @@ supports customers that want to:
...
+It is built by the CI on any changes to the frontend codebase.
```
Comment (ivakoleva) [inline specs/vep-1507-vdk-operations-ui/README.md:145]: How does the frontend application image requirements differ from the frontend test image already produced?

File: specs/vep-1507-vdk-operations-ui/README.md [hunk 3]
```diff
@@ -138,6 +140,16 @@ supports customers that want to:
...
+
+#### Helm(or install helpers)
+There is no use case for having just the frontend installed.
+Because of this there are no helpers for installing the frontend.
+Instead, the frontend will be installed as part of the contrl-service helm chart
```
Comment (ivakoleva) [inline specs/vep-1507-vdk-operations-ui/README.md:150]: nit: may link to the chart

File: specs/vep-1507-vdk-operations-ui/README.md [hunk 4]
```diff
@@ -138,6 +140,16 @@ supports customers that want to:
 1. Use the shared component libraries in their projects.
 2. Build their own docker image for data-pipelines
 
+#### Docker
+A Docker image will be published on docker hub under the user https://hub.docker.com/u/versatiledatakit.
+It is built by the CI on any changes to the frontend codebase.
+
+#### Helm(or install helpers)
+There is no use case for having just the frontend installed.
```
Comment (ivakoleva) [inline specs/vep-1507-vdk-operations-ui/README.md:148]: Is there? And an use case where only the backend is installed?

File: specs/vep-1507-vdk-operations-ui/README.md (hunk 2 above)
Comment (murphp15) [reply specs/vep-1507-vdk-operations-ui/README.md:145]: The frontend test image includes browsers and testing utils. 
This is completely needless bloat for the production system. 
In the production image we just include the distribution folder from the build directory and nginx server.

File: specs/vep-1507-vdk-operations-ui/README.md (hunk 4 above)
Comment (murphp15) [reply specs/vep-1507-vdk-operations-ui/README.md:148]: I think there is an argument for only installing the backend 
1. if customers wanted to use their own UI (seems very unlikely)
2. if customers wanted to use just the cli tools, vdk-cli,curl etc... 



We can make the frontend ui configurable similar to how cockroach db is configurable.

File: specs/vep-1507-vdk-operations-ui/README.md (hunk 2 above)
Comment (ivakoleva) [reply specs/vep-1507-vdk-operations-ui/README.md:145]: May add the requirements known (about the image) to the tickets for creating the runtime image.

**Pull request closed without merging**